doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -192,6 +198,48 @@ analysis on inductive or coinductive objects (see :ref:`variants`).
      Makes the tactic equivalent to
      :tacn:`induction` :n:`{+, @induction_clause } @induction_principle`.
 
+   .. example:: Using :tacn:`destruct` on the conclusion
+
+      Creates a subgoal for each constructor, substituting the constructor
+      into the conclusion.
+
+      .. rocqtop:: reset none
+
+         Goal forall m n: nat, m + n = n + m.
+
+      .. rocqtop:: out
+
+         intros.
+
+      .. rocqtop:: all
+
+         destruct n.   (* n is an inductive *)
+
+   .. _example_split_hypothesis:
+
+   .. example:: Using :tacn:`destruct` on a hypothesis
+
+      This gives the effect of a :tacn:`split` or :tacn:`constructor` on the
+      hypothesis.  Creates hypotheses for each constructor of the head constant.
+
+      .. rocqtop:: reset none
+
+         Goal forall A B: Prop, A /\ B -> True.
+
+      .. rocqtop:: out
+
+         intros.
+
+      .. rocqtop:: all
+
+         destruct H.   (* H is a hypothesis *)
+
+   .. For this case, destruct H simply removes H.  That doesn't seem expected/useful.
+      Maybe should not do this?
```

Comment:
That's the expected outcome of doing a pattern match on an equality where the index doesn't match any part of the conclusion or any other hypothesis. This may deserve an explanation somewhere, but it is intended.

Reply:
The purpose of the comment is to note the behavior for the benefit of those who update the documentation.  It doesn't appear in the formatted documentation.

Reply:
I'm not sure what benefit those who update the documentation would get from this comment; hence my message. I get that it's a comment, but I still find it misleading for saying that destruct H simply removing H seems unexpected or useless, when it's entirely expected and can be useful (although I admit, in rare circumstances).